sycl/include/CL/sycl/INTEL/esimd/detail/emu/esimdcpu_device_interface.hpp

```diff
@@ -0,0 +1,37 @@
+//==----- esimdcpu_device_interface.hpp - DPC++ Explicit SIMD API ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+/// \file esimdcpu_device_interface.hpp
+/// Declarations for ESIMD_CPU-device specific definitions.
+/// ESIMD intrinsic and LibCM functionalities required by intrinsic defined
+///
+/// \ingroup sycl_pi_esimd_cpu
+
+#pragma once
+
+#include <cstdint>
+
+#ifdef _MSC_VER
+typedef unsigned int uint;
+typedef unsigned short ushort;
+typedef unsigned char uchar;
+#endif // _MSC_VER
```

Comment:
Could you please clarify why those typedef's are needed?

Reply:
Will do.